mifos-payment/src/main/java/org/mifos/mifospaymentbridge/model/Category.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * File: Category.java
+ * =========================================
+ * This class models the category
+ * table in our database.
+ * @author Vladimir Fomene
+ **/
+
+package org.mifos.mifospaymentbridge.model;
+
+import org.joda.time.DateTime;
+import javax.persistence.Column;
+import javax.persistence.Table;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Table(name = "category")
+public class Category {
+
+    /**
+     * table Fields of the category table
+     */
+    @Id
+    @GeneratedValue(strategy=GenerationType.AUTO)
+    @Column(name="category_id")
+    private Integer id;
+
+    @Column(name="category_name")
+    private String categoryName;
+
+    @Column(name="last_modified_dtm")
+    private DateTime lastModifiedDtm;
```

Comment:
I decided to use joda because it has a DateTime type which correspond to the type of lastModifiedDtm field and java doesn't. It also has more DateTime manipulation functions that can come in handy.